formats/application/vnd/gs1/produce/receiving_event/1/+json/example/default.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+
+  _type: 'application/vnd.gs1.produce.receiving_event.1+json',
```

Comment:
I think we should avoid using the `gs1` top-level vendor prefix in media types since that should probably be left to GS1 themselves.  I'd suggest using `application/vnd.trellis.gs1...` to keep it under our vendor prefix.

Reply:
sorry for the delayed response, I also thought of it, I felt if we started using Gs1 as a top level that would attract them here. Adding it under Trellis would make it restricted - what do you think?  If not I can put it under  Trellis.